README.md

```diff
@@ -43,7 +44,8 @@ mindee_client = Client(
 )
 ```
 
-We suggest storing your credentials as environment variables. Why? You'll never have to worry about committing your credentials and accidentally posting them somewhere public.
+We suggest storing your credentials as environment variables.
+Why? You'll never have to worry about committing your credentials and accidentally posting them somewhere public.
```

Comment:
Can we write how to do this or point them to a documentation on how to do that?

Reply:
This is a feature parity problem actually, since the nodejs SDK has native envvar support.

We were planning on adding this for v2 with the rework.

Reply:
oh, but in the meantime, when they use the code to use the SDK, they can have that code using env variable. I thought it's what we were referring to here.

Reply:
Yes they could, but pointless to document now since in another week or so we'll have native support. It takes very little time to add it.

Reply:
Oh ok, I had no idea v2 was in a week, sounds good!